modules/hibernating-cluster-hibernate.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-hibernate_{context}"]
+= Hibernating a cluster
+
+You can hibernate a cluster for up to 90 days. The cluster can recover if certificates expire while the cluster was in hibernation.
+
+.Prerequisites
+
+* The cluster has been running for at least 24 hours to allow the first certificate rotation to complete.
++
+If you must hibernate your cluster before the 24 hour certificate rotation, then use the following procedure instead: link:https://www.redhat.com/en/blog/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms[Enabling OpenShift 4 Clusters to Stop and Resume Cluster VMs].
+
+* You have access to the cluster as a user with the `cluster-admin` role.
++
+// TODO: Confirm must be a cluster-admin
+
+.Procedure
+
+. Take an etcd backup before hibernating your cluster. For instructions, see "Backing up etcd".
++
+// TODO: Do we need a warning like the graceful shutdown has? If something goes wrong, will they follow the steps to "restore to a previous cluster state"?
```

Comment:
@vrutkovs Do you think we need a warning like we have here: https://docs.openshift.com/container-platform/4.17/backup_and_restore/graceful-cluster-shutdown.html#prerequisites

Would it be true that if something went wrong during hibernating/resuming, that they'd follow the restoring to a previous cluster state procedure?